### DISCUSSION TITLE
Modify vehicle category coefficients

### DISCUSSION THREAD

**Discussion by lyselott** [Universite-Gustave-Eiffel/NoiseModelling#606] [Q&A] [answered]
Hi again,

- How can I modify the cat 1, 2, 3, 4a and 4b vehicles coefficients? (I have found something in RoadSourceParametersCnossos.java but I don't know how to personalize it) or

- Is it possible to include the 5 cat vehicle? (with new coefficients, ex. from measurements)

Best regards,
Alexandra

**Comment by pierromond** [comment, marked as answer]
Dear @lyselott,

Yes you can, you juste have to change this file : https://github.com/Universite-Gustave-Eiffel/NoiseModelling/blob/4.X/noisemodelling-emission/src/main/resources/org/noise_planet/noisemodelling/emission/road/cnossos/RoadCnossos_2020.json

But then you will need to compile again the software : here a tutorial that we gave last year to do so, I hope it can help you ! https://filesender.renater.fr/?s=download&token=f4176f86-a111-4852-85e2-382d7619c0ca

Best regards.

P.S. If you succeed to do it, and if your coefficient are free to share, it would be very nice if you can share your new file with the community !

**Reply by lyselott** [reply]
Thank you Pierre, could you please check the tutorial link? This does not work.

Regards!

**Reply by matlendzi** [reply]
+1 Could someone provide the tutorial again? We would possibily need to adjust the road surface type parameters to adjust to German RLS 19.

**Reply by nicolas-f** [reply]
Hello,

You can fork the repository, edit the json file then github action will deliver you your own noisemodelling_without_gui.zip file. (in **Actions** tab)

This is not part of the tutorial, we will add this in a guide into the documentation. Editing on github is by far the faster method.
